set.mm

```diff
@@ -420312,6 +420323,35 @@ the theorem forms ("i", "d" or "g") should be the first of multiple
        </p>
 
        <ul>
+       <li><b>Contributors.</b>
+       The (original) contributor of a theorem is the first one who proved the
+       theorem and added it to set.mm. She/he is mentioned in the "Contributed
+       by" parenthetical within the comment of the theorem, see Metamath Book,
+       p. 142.
+       An exception should be made if a theorem is an (essential) extract or a
```

Comment:
I am not sure whether this is too general.  I would like to hear how a case like [wl-2sb6d](file:///home/wolf/metamath/web/wl-2sb6d.html) is handled.

Reply:
For instance in wl-2sb6d, the differences in the proof look substantial enough that you remain the contributor. If it were to *replace* 2sb6 by Norm, then there would be no credit to Norm left, so you could write in the comment of 2sb6d something like "replaces an earlier version contributed by NM on ..."

Reply:
Your reaction to this real-world example points out there is somewhere a zone of uncertainty, where decisions flip and people may not always agree.  Okay.  Its fine to take a risk, as long as you are aware of it.